ui/perfherder/graphs/TableView.jsx

```diff
@@ -101,44 +161,11 @@ const TableView = ({
         </span>
       ),
       headerClassName: `text-wrap d-flex justify-content-center align-items-center table-header ${item.color[0]}`,
-      Cell: (props) => {
-        let cellElem = null;
-        if (props.original[dataKey]) {
-          const {
-            value,
-            jobsUrl,
-            compareUrl,
-            deltaValue,
-            deltaPercent,
-          } = props.original[dataKey];
-          cellElem = (
-            <div tabIndex={-1}>
-              <span>
-                {value}{' '}
-                <span className="text-darker-secondary">
-                  &Delta; {displayNumber(deltaValue.toFixed(1))} (
-                  {(100 * deltaPercent).toFixed(1)}
-                  %)
-                </span>
-              </span>
-              <br />
-              <div className="job-links">
-                {jobsUrl && (
-                  <a href={jobsUrl} target="_blank" rel="noopener noreferrer">
-                    job
-                  </a>
-                )}
-                ,{' '}
-                <a href={compareUrl} target="_blank" rel="noopener noreferrer">
-                  compare
-                </a>
-              </div>
-            </div>
-          );
-        }
-        return cellElem;
-      },
-      getProps: (state, rowInfo) => setHighlightedRow(rowInfo),
+      Cell: TestCell,
+      getProps: (state, rowInfo) => ({
+        ...setHighlightedRow(rowInfo),
+        dataKey,
+      }),
```

Comment:
I couldn't define the PropTypes for `props` of the anonymous arrow function, so I had to extract it to a separate named function following the suggestion from https://github.com/yannickcr/eslint-plugin-react/issues/2404#issuecomment-529232335